<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>App Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="topMargin">
         <number>7</number>
        </property>
        <property name="verticalSpacing">
         <number>7</number>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="quickResultCheckBox">
          <property name="toolTip">
           <string>Open translator from anywhere on system</string>
          </property>
          <property name="text">
           <string>Quick Translate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Theme</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QKeySequenceEdit" name="keySequenceEdit"/>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="dark">
            <property name="text">
             <string>Dark</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="light">
            <property name="text">
             <string>Light</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>About</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextBrowser" name="aboutTextBrowser">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>215</height>
           </size>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="controlButton" name="donate">
            <property name="toolTip">
             <string>Donate via Paypal</string>
            </property>
            <property name="text">
             <string>Donate</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/paypal-line.png</normaloff>:/icons/paypal-line.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="controlButton" name="github">
            <property name="toolTip">
             <string>Star repository on github</string>
            </property>
            <property name="text">
             <string>Github</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/links-line.png</normaloff>:/icons/links-line.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="controlButton" name="rate">
            <property name="toolTip">
             <string>Rate this app in Software App</string>
            </property>
            <property name="text">
             <string>Rate</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/star-line.png</normaloff>:/icons/star-line.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>controlButton</class>
   <extends>QPushButton</extends>
   <header>controlbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
